build_tools/third_party/tracy/CMakeLists.txt

```diff
@@ -115,7 +115,7 @@ endfunction()
 # Common library
 #-------------------------------------------------------------------------------
 
-file(GLOB COMMON_SRCS ${TRACY_SOURCE_DIR}/common/*.cpp)
+file(GLOB COMMON_SRCS ${TRACY_SOURCE_DIR}/public/common/*.cpp)
```

Comment:
Not sure if this is a flake, but something in Tracy errored on our release build: https://github.com/iree-org/iree/runs/7629125198?check_suite_focus=true

```
  [274/348] Linking CXX executable tools/iree-run-module
  [275/348] Linking CXX executable tools/iree-run-trace
  [276/348] Building CXX object tracy/CMakeFiles/IREETracyCommon.dir/__/__/__/third_party/tracy/public/common/TracyStackFrames.cpp.o
  [277/348] Building CXX object tracy/CMakeFiles/IREETracyCommon.dir/__/__/__/third_party/tracy/public/common/TracySystem.cpp.o
  [278/348] Building CXX object tracy/CMakeFiles/IREETracyCommon.dir/__/__/__/third_party/tracy/public/common/TracySocket.cpp.o
  [279/348] Building CXX object runtime/bindings/python/CMakeFiles/iree_runtime_bindings_python_PyExtRt.dir/vm.cc.o
  In file included from /main_checkout/iree/runtime/bindings/python/vm.cc:7:
  /main_checkout/iree/runtime/bindings/python/./vm.h: In static member function ‘static void iree::python::VmRef::BindRefProtocol(PyClass&, TypeIdFunctor, RetainRefFunctor, DerefFunctor, IsaFunctor)’:
  /main_checkout/iree/runtime/bindings/python/./vm.h:213:11: warning: typedef ‘using RawPtrType = typename PyClass::type::RawPtrType’ locally defined but not used [-Wunused-local-typedefs]
    213 |     using RawPtrType = typename WrapperType::RawPtrType;
        |           ^~~~~~~~~~
  [280/348] Building CXX object tracy/CMakeFiles/IREETracyCommon.dir/__/__/__/third_party/tracy/public/common/tracy_lz4hc.cpp.o
  [281/348] Building CXX object tracy/CMakeFiles/IREETracyServer.dir/__/__/__/third_party/tracy/server/TracyColor.cpp.o
  [282/348] Building CXX object tracy/CMakeFiles/IREETracyCommon.dir/__/__/__/third_party/tracy/public/common/tracy_lz4.cpp.o
  [283/348] Linking CXX static library tracy/libIREETracyCommon.a
  [284/348] Building CXX object tracy/CMakeFiles/IREETracyServer.dir/__/__/__/third_party/tracy/server/TracyBadVersion.cpp.o
  [285/348] Building CXX object tracy/CMakeFiles/IREETracyServer.dir/__/__/__/third_party/tracy/server/TracyEventDebug.cpp.o
  FAILED: tracy/CMakeFiles/IREETracyServer.dir/__/__/__/third_party/tracy/server/TracyEventDebug.cpp.o
  /opt/rh/devtoolset-10/root/usr/bin/c++  -I/main_checkout/iree/build_tools/third_party/tracy/../../../third_party/tracy/imgui -I/usr/include/capstone -I/usr/include/capstone/capstone -O3 -DNDEBUG -fPIC -Wno-unused-result -std=gnu++17 -MD -MT tracy/CMakeFiles/IREETracyServer.dir/__/__/__/third_party/tracy/server/TracyEventDebug.cpp.o -MF tracy/CMakeFiles/IREETracyServer.dir/__/__/__/third_party/tracy/server/TracyEventDebug.cpp.o.d -o tracy/CMakeFiles/IREETracyServer.dir/__/__/__/third_party/tracy/server/TracyEventDebug.cpp.o -c /main_checkout/iree/third_party/tracy/server/TracyEventDebug.cpp
  /main_checkout/iree/third_party/tracy/server/TracyEventDebug.cpp:6:10: fatal error: ../common/TracyQueue.hpp: No such file or directory
      6 | #include "../common/TracyQueue.hpp"
        |          ^~~~~~~~~~~~~~~~~~~~~~~~~~
  compilation terminated.
  [286/348] Building CXX object tracy/CMakeFiles/IREETracyServer.dir/__/__/__/third_party/tracy/server/TracyImGui.cpp.o
  [287/348] Linking CXX shared module runtime/bindings/python/iree/_runtime.cpython-37m-x86_64-linux-gnu.so
  ninja: build stopped: subcommand failed.
```

Reply:
Ah, looks like we only set `-DIREE_BUILD_TRACY=ON` in the releases, and that is failing.